tests/functional/aws-node-sdk/test/multipleBackend/mpuComplete/azureCompleteMPU.js

```diff
@@ -0,0 +1,250 @@
+                Bucket: azureContainerName,
+                Key: key,
+                PartNumber: 2,
+                UploadId: uploadId,
+                Body: bigBody,
+            };
+            s3.uploadPart(partParams, (err, res) => {
+                partArray.push({ ETag: res.ETag, PartNumber: 2 });
+                next(err, uploadId);
+            });
+        },
+    ], (err, uploadId) => {
+        process.stdout.write('Created MPU and put two parts\n');
+        assert.equal(err, null, `Err setting up MPU: ${err}`);
+        cb(uploadId, partArray);
+    });
+}
+
+describeSkipIfNotMultiple('Complete MPU API for Azure data backend',
+function testSuite() {
+    this.timeout(150000);
+    withV4(sigCfg => {
+        beforeEach(function beFn() {
+            this.currentTest.key = `somekey-${Date.now()}`;
+            bucketUtil = new BucketUtility('default', sigCfg);
+            s3 = bucketUtil.s3;
+            return s3.createBucketAsync({ Bucket: azureContainerName })
+            .catch(err => {
+                process.stdout.write(`Error creating bucket: ${err}\n`);
+                throw err;
+            });
+        });
+
+        afterEach(() => {
+            process.stdout.write('Emptying bucket\n');
+            return bucketUtil.empty(azureContainerName)
+            .then(() => {
+                process.stdout.write('Deleting bucket\n');
+                return bucketUtil.deleteOne(azureContainerName);
+            })
+            .catch(err => {
+                process.stdout.write(`Error in afterEach: ${err}\n`);
+                throw err;
+            });
+        });
+
+        it('should complete an MPU on Azure', function itFn(done) {
+            mpuSetup(this.test.key, azureLocation, (uploadId, partArray) => {
+                const params = {
+                    Bucket: azureContainerName,
+                    Key: this.test.key,
+                    UploadId: uploadId,
+                    MultipartUpload: { Parts: partArray },
+                };
+                s3.completeMultipartUpload(params, err => {
+                    assert.equal(err, null, `Err completing MPU: ${err}`);
+                    setTimeout(() => getCheck(this.test.key, done),
+                        azureTimeout);
+                });
+            });
+        });
+
+        it('should complete an MPU on Azure with bucketMatch=false',
+        function itFn(done) {
+            mpuSetup(this.test.key, azureLocationMismatch,
+            (uploadId, partArray) => {
+                const params = {
+                    Bucket: azureContainerName,
+                    Key: this.test.key,
+                    UploadId: uploadId,
+                    MultipartUpload: { Parts: partArray },
+                };
+                s3.completeMultipartUpload(params, err => {
+                    assert.equal(err, null, `Err completing MPU: ${err}`);
+                    setTimeout(() => getCheck(this.test.key, done),
```

Comment:
Is the test failing here? 
On the Azure side, your blob should not be inside `azureContainerName` container with blob name equals to `azureContainerName/this.test.key`?